IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo

```diff
@@ -123,12 +124,14 @@ equation
   connect(senTem.port_b,loa. ports[1])
     annotation (Line(points={{210,-60},{222,-60}},
                                                  color={0,127,255}));
-  connect(valTop.port_a, tan.fluPorVol[1]) annotation (Line(points={{112,-20},{-112.6,
-          -20},{-112.6,-120}}, color={0,127,255}));
-  connect(valMid.port_a, tan.fluPorVol[3]) annotation (Line(points={{132,-60},{-112.6,
-          -60},{-112.6,-120}},color={0,127,255}));
-  connect(valBot.port_a, tan.fluPorVol[5]) annotation (Line(points={{150,-100},{
-          -112.6,-100},{-112.6,-120}},
+  connect(valTop.port_a, tan.fluPorVol[1]) annotation (Line(points={{112,-20},{
+          -116,-20},{-116,-118},{-112.6,-118},{-112.6,-118.4}},
+                               color={0,127,255}));
+  connect(valMid.port_a, tan.fluPorVol[3]) annotation (Line(points={{132,-60},{
+          -116,-60},{-116,-118},{-112.6,-118}},
+                              color={0,127,255}));
+  connect(valBot.port_a, tan.fluPorVol[5]) annotation (Line(points={{150,-100},
+          {-116,-100},{-116,-118},{-112.6,-118},{-112.6,-117.6}},
```

Comment:
@mwetter Just a question: Why doesn't git show the real diff? It looks like functional changes at the first glance, but only the annotation changed. Some save-settings in Dymola?